src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -100,4 +100,11 @@ function Base.show(io::IO, ::MIME"text/plain", limiter::SubcellLimiterIDP)
         summary_box(io, "SubcellLimiterIDP", setup)
     end
 end
+
+function get_node_variables!(node_variables, limiter::SubcellLimiterIDP,
+                             ::VolumeIntegralSubcellLimiting, equations)
+    node_variables[:alpha_limiter] = limiter.cache.container_subcell_limiter.alpha
+    # Note: alpha is not filled before the first timestep. (TODO?)
```

Comment:
Is it acceptable to have undefined alphas before the first time step?

Reply:
Do you mean by "undefined" that the array was allocated but not yet filled with values? Maybe you can check what happens for the Hennemann Gassner indicator, whether it is also undefined for the first file.

However, IIRC, in one of the callbacks we make at least one call to `rhs!` during initialization (maybe `AnalysisCallback`?). Thus in that case you should already get some alpha values for the initial condition.

Reply:
Yes, exactly. I export values like `1e306` for the first file.
Yes, `rhs!` is called once before for some callback. That's why the coefficient for the HennemannGassner indicator got useful values.
Unfortunately, the coefficients for the IDP limiting are not calculated inside `rhs!` but in the a posteriori limiting phase, implemented as stage callback SubcellLimitingCorrection.
I somehow have to call the limiter itself somewhere in the beginning. Before the SaveSolution data export, but after `rhs!` since the antidiffusive fluxes are calculated in there, which are needed to calculate the blending factors.

Alternatively, I could keep it as it is or set them to zero for the first file.

Reply:
Go for the easiest approach for now (uninitialized). Possibly refine later. That's what I would recommend